### ISSUE TITLE
OT: make Internet slang and Emoticons into npm package

### ISSUE THREAD

**Issue by dandv** [Zenexer/tg-help#3]
Sorry for the OT issue, but [Gist doesn't support comments](https://github.com/isaacs/github/issues/21#issuecomment-239340234).

I found your [Gist about emoticons and chat abbreviations](https://gist.github.com/Zenexer/af4dd767338d6c6ba662) and was wondering if you'd like to make it into a package. Existing packages are very old, or don't cover many of the terms in your gist.

* https://www.npmjs.com/package/slang-translator
* https://www.npmjs.com/package/fin-slang
* https://www.npmjs.com/package/emoji-text
* https://www.npmjs.com/package/smslingo

**Comment by Zenexer** [comment]
@dandv Sorry, I never noticed the alert for this.  Would you still have any interest in such a package?

**Comment by dandv** [comment]
Yes, if the npm ecosystem situation is the same. I did find a newer package than the ones I listed in 2018 though, https://www.npmjs.com/package/emoji-translator
